### PR TITLE
Vault config

### PR DESCRIPTION
**Pull request by adamgraves-choices**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,42 +2,64 @@
 
 This repo contains deployment scripts executed during project builds on [Travis CI](https://travis-ci.org) and [Gitlab CI](https://about.gitlab.com/features/gitlab-ci-cd/)
 
-The scripts should work across a number of repos that include a `Dockerfile` and a `docker-compose.production.yml` file. They are intended to be included as a [Git submodule](https://git-scm.com/docs/git-submodule) within the code repo.
+The scripts should work across a number of repos that include a `Dockerfile` and a `docker-compose.yml` file within the `/rancher-config/` directory. They are intended to be included as a [Git submodule](https://git-scm.com/docs/git-submodule) within the code repo.
 
-Including this repo within your own (as detailed below) will:
+Including this repo within your re (as detailed below) will:
```
Comment (markysoft) [inline README.md]: not sure why this text changed, should it say 'Including this repo within your own repo'

**Review by markysoft — DISMISSED** [review]
I don't know enough abut the scripts to comment on the implementation, but have checked the readme which is good aside from a small typo

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,10 @@
 sudo: required
 
+addons:
```
Comment (st3v3nhunt) [inline .travis.yml:5]: `jq` should be available in the images by default. I thought I'd seen it mentioned in the docs but I'm unable to find it. However, this comment confirms it https://github.com/travis-ci/travis-ci/issues/7954#issuecomment-310527961

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]
I know this works for Travis as I've tried it with the `frontend-library`. I've submitted a PR for changes to the wording in the `README`.

**Review by st3v3nhunt — APPROVED** [review]
Tested the changes with the `frontend-library` https://github.com/nhsuk/frontend-library/pull/5 and all worked ok.

**Pull request merged**